API/src/main/java/io/github/jwdeveloper/tiktok/data/models/gifts/Gift.java

```diff
@@ -31,6 +29,22 @@ public Gift(int id, String name, int diamondCost, String pictureLink, JsonObject
         this.properties = properties;
     }
 
+    public Gift(int id, String name, int diamondCost, String pictureLink) {
+        this.id = id;
```

Comment:
Switch to `this(id, name, diamondCost, new Picture(pictureLink))`

Reply:
done